readme.md

```diff
@@ -25,6 +25,8 @@ $ git add .
 $ git commit -m "packaged env"
 $ git push
 ```
+### [metang] can we make git add/commit part of hub package-env? it's easier to have hub, the push to get the environment checked-in
+
```

Comment:
I think it might be better to let Git do its job since people are more familiar with it?